cspell.json

```diff
@@ -21,6 +22,7 @@
         "AMPL",
         "Ampleforth",
         "anchorme",
+        "anomynous",
```

Comment:
`anomynous` is a real typo for `anonymous`. You should you inline ignore instead.